liteshort.py

```diff
@@ -121,6 +123,14 @@ def check_self_link(long):
     return False
 
 
+def linking_to_blacklist(long):
+    # Removes protocol and other parts of the URL to extract the domain name
+    long = long.split("//")[-1].split("/")[0]
+    if long in current_app.config["blacklist"]:
+        return True
+    return False
```

Comment:
You can replace this with `return long in current_app.config["blacklist"]`

Reply:
True, my bad for oversighting this.